src/components/carousel/Carousel.vue

```diff
@@ -159,11 +156,11 @@ export default {
     },
     methods: {
         startTimer() {
-            if (!this.autoplay || this.interval <= 0 || this.timer) return
+            if (!this.autoplay || this.timer) return
             this.isPause = false
             this.timer = setInterval(() => {
                 this.next()
-            }, this.interval)
+            }, (this.interval || config.defaultCarouselInterval))
```

Comment:
what about situation when interval is 0?

Reply:
@afwn90cj93201nixr2e1re you shouldn't set it to zero

Reply:
@afwn90cj93201nixr2e1re Then back to use default config, this also work and then why you use interval 0?

Reply:
Why not? immediately change on some event.

Reply:
Interval of 0 would not make any sense for autoplay.

Reply:
I know. Read https://github.com/buefy/buefy/pull/1997#discussion_r348090273 again.

Reply:
Yeah, I read it and it does not make sense anyway. The interval is for the autoplay feature. Not for some event.

Reply:
Seems like props which are reactive is nothing for you. :/